src/fightPage/buttonsEventListeners.ts

```diff
@@ -57,13 +72,28 @@ export const attacksButtonEventListener = (gameHandler: GameHandler) => {
   )! as HTMLCollectionOf<HTMLElement>;
   const backButton = document.querySelector("#backButton")! as HTMLDivElement;
 
+  const moveOne = gameHandler.currentPlayer.getActivePokemon.moves[0];
+  const moveTwo = gameHandler.currentPlayer.getActivePokemon.moves[1];
   animationButtonsEntry(battleButtons);
-  magicFunction(attackButtonOne, battleButtons, gameHandler, attack);
-  magicFunction(attackButtonTwo, battleButtons, gameHandler, attack);
+  magicFunction(attackButtonOne, battleButtons, gameHandler, attack, moveOne);
```

Comment:
```
const pokemon = gameHandler.currentPlayer.getActivePokemon;
const move = pokemon.moves.find(move => move.moveName === e.currentTarget.textContent
```
Maybe instead of adding moves to magic funtion and event listeners, you could use the following code in attack function? (pass event as parameter to the attack function?)

Reply:
ok, I used getting move by event.
I will merge it after you pull request will be on master.